src/models/search/Debug.php

```diff
@@ -92,20 +93,20 @@ public function attributeLabels()
      */
     public function search($params, $models)
     {
-        $dataProvider = new ArrayDataProvider([
+        $dataProvider = Yii::createObject([
```

Comment:
Why?

Reply:
Via constructor can be now set only modue id (not configurable array) and configure new `ArrayDataProvider` can be done via `Yii::createObject` or ArrayDataProvider'setters.

Reply:
Do we need it to be configurable?

Reply:
At least you need set data for provider and sorting attributes. In other models is used the same approach: `Yii::createObject`

Reply:
OK.